### PR TITLE
Alerting: Add plugins extension point to alerting home page

### PR DESCRIPTION
**Pull request by konrad147**
**What is this feature?**
This PR adds a new extension point in the Alerting homepage where plugins creating own alerts can insert their components

**Why do we need this feature?**
To provide better Alerting integration with plugins having their own alert rules

**Who is this feature for?**

Fixes #

**Special notes for your reviewer:**

image

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by soniaAguilarPeiron — APPROVED** [review]
LGTM! Nice work!

File: public/app/features/alerting/unified/home/GettingStarted.tsx [hunk 1]
```diff
@@ -3,23 +3,8 @@ import React from 'react';
 import SVG from 'react-inlinesvg';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { EmbeddedScene, SceneFlexLayout, SceneFlexItem, SceneReactObject } from '@grafana/scenes';
 import { useStyles2, useTheme2, Stack, Text, TextLink } from '@grafana/ui';
 
-export const getOverviewScene = () => {
```
Comment (tomratcliffe) [inline public/app/features/alerting/unified/home/GettingStarted.tsx:9]: (I need to learn more about Scenes in general, where can I do that? I don't understand the impact of removing this here)

**Review by tomratcliffe — COMMENTED** [review]
We've lost the padding between the tabs and the content above/below, BTW:
image


Otherwise code looks good!

File: public/app/features/alerting/unified/home/GettingStarted.tsx (hunk 1 above)
Comment (konrad147) [reply public/app/features/alerting/unified/home/GettingStarted.tsx:9]: I guess this is the best place https://grafana.com/developers/scenes/

**Review by tomratcliffe — APPROVED** [review]

**Pull request merged**